code/iaas/logic/src/main/java/io/cattle/platform/process/containerevent/ContainerEventCreate.java

```diff
@@ -159,10 +199,10 @@ void setImage(Map<String, Object> inspect, Instance instance) {
         String imageUuid = (String)config.get(INSPECT_IMAGE);
 
         // Somewhat of a hack, but needed for testing against sim contexts
-        if ( !imageUuid.matches(IMAGE_KIND_PATTERN) ) {
+        if (!imageUuid.matches(IMAGE_KIND_PATTERN)) {
             imageUuid = IMAGE_PREFIX + imageUuid;
         }
-        DataAccessor.fields(instance).withKey(InstanceConstants.FIELD_IMAGE_UUID).set(imageUuid);
+        DataAccessor.fields(instance).withKey(FIELD_IMAGE_UUID).set(imageUuid);
         Image image = null;
         try {
             image = storageService.registerRemoteImage(imageUuid);
```

Comment:
@ibuildthecloud 
How does io.cattle.platform.core.dao.StoragePoolDao.findExternalActivePools() get scoped to a particular account? It doesn't have an account id in the query.

Called here
https://github.com/rancherio/cattle/blob/c552571c3d2003446cc5af9269563cc4da3258db/code/iaas/storage-service/src/main/java/io/cattle/platform/storage/service/impl/StorageServiceImpl.java#L38